docs/creating-new/autotools.rst

```diff
@@ -0,0 +1,99 @@
+Autotools
+=========
+
+Very often, you will come across a package that uses autotools as its build
+system and does not support CMake builds. Although Hunter prefers CMake builds
+when possible, it does support autotools projects when no CMake build is
+available. Here is how to do it.
+
+.. code-block:: cmake
+
+    # !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+    include(hunter_add_version)
+    include(hunter_configuration_types)
+    include(hunter_pick_scheme)
+    include(hunter_download)
+    include(hunter_cacheable)
+    include(hunter_cmake_args)
+
+    hunter_add_version(
+        PACKAGE_NAME
+        foo
+        VERSION
+        "1.2.3"
+        URL
+        "https://example.com/foo-1.2.3.tar.gz"
+        SHA1
+        da39a3ee5e6b4b0d3255bfef95601890afd80709
+    )
+
+    # More versions...
+
+    # Optional platform customization.
+    if (ANDROID OR IOS)
+      hunter_cmake_args(
+          foo
+          CMAKE_ARGS
+              EXTRA_FLAGS=--enable-x
+      )
+    endif()
+
+    hunter_configuration_types(foo CONFIGURATION_TYPES Release)
+    hunter_pick_scheme(DEFAULT url_sha1_autotools)
+    hunter_cacheable(foo)
+    hunter_download(PACKAGE_NAME foo)
+
+Note that the build may not be cacheable if autotools generation expands
+absolute paths. Try using ``hunter_cacheable`` and see if it works.
+
+**Important**: by default, this build will not provide any CMake config.
+Therefore, projects using autotools **must provide a file ``Findfoo.cmake`` in
+the ``cmake/find`` directory**. Dependent projects will use code similar to the
+following:
+
+.. code-block:: cmake
+
+    hunter_add_package(foo)
+    find_package(foo REQUIRED)
+    add_executable(bar ${BAR_SOURCES})
+    target_include_directories(bar PUBLIC ${FOO_INCLUDE_DIRS})
```

Comment:
I'm okay with something like this in example but not in documentation. `Findfoo.cmake` should provide imported target instead of `FOO_{INCLUDE_DIRS,LIBRARIES}`. I.e. it should be something like `target_link_libraries(bar PUBLIC foo::foo)`.